src/API/Plugin.php

```diff
@@ -0,0 +1,25 @@
+<?php
+
+namespace CF\API;
+
+class Plugin extends Client
+{
+    const PLUGIN_API_NAME = 'PLUGIN API';
```

Comment:
instead of PLUGIN_API_NAME, CLIENT_API_NAME, HOST_API_NAME we can use API_NAME to represent all of them in AbstractAPIClient.php